nautobot/extras/models/models.py

```diff
@@ -645,6 +645,48 @@ def size(self):
             return None
 
 
+#
+# Notes
+#
+
+
+@extras_features("graphql", "webhooks")
+class Notes(BaseModel, ChangeLoggedModel):
+    """
+    Notes allow anyone with proper permissions to add a note to an object.
+    """
+
+    assigned_object_type = models.ForeignKey(to=ContentType, on_delete=models.CASCADE)
+    assigned_object_id = models.UUIDField(db_index=True)
+    assigned_object = GenericForeignKey(ct_field="assigned_object_type", fk_field="assigned_object_id")
+    user = models.ForeignKey(
+        to=settings.AUTH_USER_MODEL,
+        on_delete=models.SET_NULL,
+        related_name="note",
+        blank=True,
+        null=True,
+    )
+    user_name = models.CharField(max_length=150, editable=False)
+    name = models.CharField(max_length=150)
+    slug = AutoSlugField(populate_from="name", max_length=150)
+    note = models.TextField()
+    objects = NotesQuerySet.as_manager()
+
+    class Meta:
+        ordering = ["created"]
```

Comment:
Just a note that `created` is currently only a DateField, not a DateTimeField, (#2076) so you may see unexpected ordering when multiple notes are created on the same day.

Reply:
Great point, I should probably change to DateTimeField then.